### PR TITLE
I can't stop I can't stop I can't stop I can't stop

### PR DESCRIPTION
**Pull request by FazanaJ**

### CODE REVIEW

File: src/objects.c [hunk 1]
```diff
@@ -109,11 +109,11 @@ s8 D_800DC820[10] = {1, 1, 1, 1, 1, 1, 1, 1, 1, 1};
...
 s8 D_800DC840[8] = { 9, 1, 2, 3, 4, 5, 7, 0 };
 
-s8 D_800DC848 = 0;
-u32 D_800DC84C[3] = {
-    0xFF401000,
-    0x1040FF00,
-    0x10FF4000,
+s8 gNoBoundsCheck = FALSE;
+u32 gMagnetColours[3] = {
+    ((255 << 24) | (64 << 16) |  (16 << 8) | 0), // Level 1
```
Comment (Ryan-Myers) [inline src/objects.c:114]: Not at a comp to check, but can this use the macro for rgba 5551? Something like Gbapack_5551 or whatever it's called?

File: src/objects.c [hunk 2]
```diff
@@ -1003,7 +1000,7 @@ void func_80010994(s32 updateRate) {
     tempVal = objCount;
     for (i = D_8011AE60; i < tempVal; i++) {
         obj = gObjPtrList[i];
-        if (!(obj->segment.trans.unk6 & 0x8000)) {
+        if (!(obj->segment.trans.flags & 0x8000)) {
```
Comment (Ryan-Myers) [inline src/objects.c:1003]: OBJ_FLAGS_DEACTIVATED? Also in other places.

**Review by Ryan-Myers — APPROVED** [review]
Lgtm

File: src/objects.c (hunk 1 above)
Comment (FazanaJ) [reply src/objects.c:114]: It's 32 bit colour, but I can probably just do another macro for this for readability reasons

**Pull request merged**